tests/op/extract/load/test_txt_op.py

```diff
@@ -0,0 +1,89 @@
+import unittest
+
+from moto import mock_s3
+
+from uniflow.node import Node
+from uniflow.op.extract.load.txt_op import ExtractTxtOp, ProcessTxtOp
+
+
+class TestExtractTxtOp(unittest.TestCase):
+    def setUp(self):
+        self.extract_txt_op = ExtractTxtOp(name="extract_txt_op")
+
+    def test_empty_input(self):
+        # arrange
+        node = Node(
+            name="node1",
+            value_dict={"filename": "tests/op/extract/data/empty.txt"},
```

Comment:
qq: this looks like a your must run the unittest in the root folder. However, people might run unittest in the sub `load` folder which might cause problem for this unittest.

Usually, I will not test I/O as well as external function call such as `read_file` in extract_txt_op. Instead, I would mock it with return_value to test method specific logic.

Reply:
Okay, I changed reading local file to mock it with return_value. 
Does `read_file` need to test I/O by reading the local file and s3 file?